db/seeds.rb

```diff
@@ -1,26 +1,25 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-
-
-ben = {
-  email: 'benjamin_c_damman@omb.eop.gov',
-  password: 'password',
-  password_confirmation: 'password'
-}
+password = ENV['DEMO_USER_PASSOWRD']
```

Comment:
how about just `SecureRandom.uuid` here, so that the user must request a reset and thereby confirm possession of the account? If the password is set in an env var, that persists yes?